manifests/n/NitrokeyGmbH/Nitropy/0.4.36.0/NitrokeyGmbH.Nitropy.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NitrokeyGmbH.Nitropy
+PackageVersion: 0.4.36.0
+PackageLocale: en-US
+Publisher: Nitrokey GmbH
+PackageName: Nitropy
+License: Apache-2.0, MIT

```

```suggestion
License: Apache-2.0, MIT
LicenseUrl: https://github.com/Nitrokey/pynitrokey#license
```
Adding LicenseUrl for better clarification of the licenses